raiden/transfer/state.py

```diff
@@ -372,9 +372,6 @@ def __post_init__(self) -> None:
         typecheck(self.open_transaction, TransactionExecutionStatus)
         typecheck(self.canonical_identifier.channel_identifier, T_ChannelID)
 
-        if self.reveal_timeout >= self.settle_timeout:
-            raise ValueError("reveal_timeout must be smaller than settle_timeout")
```

Comment:
This was an attack vector. Data from blockchain events and messages are user defined and have to be validated on a recoverable fashion, this has to be treated as tainted data